src/lib/polygon.js

```diff
@@ -0,0 +1,240 @@
+/**
+* Copyright 2012-2015, Plotly, Inc.
```

Comment:
Merge master (after https://github.com/plotly/plotly.js/pull/137), run `npm run header` and commit only the `src/` directory to update the 2015 for 2016.